### ISSUE TITLE
Stream in Home Assistant App

### ISSUE THREAD

**Issue by BeHappyYou** [s60sc/ESP32-CAM_MJPEG2SD#431]
Hello,

Is it possible to use a (picture) card to have a stream of the ESP32 camera visible in the Home Assistant App.
When I use http://IP_ADDRESS/sustain?stream=0 I see the stream when I am at home.

I would like to see it also in the Home Assistant App when I am not at home. Anyone an idea how to solve this issue?

Thanks!

**Comment by s60sc** [comment]
[port forward](https://github.com/s60sc/ESP32-CAM_MJPEG2SD#:~:text=7%3Bfps%3D1-,Port%20Forwarding,-To%20access%20the)

**Comment by BeHappyYou** [comment]
@s60sc Thank you!

I did look further into it to see if it is possible to prevent ‘port forward’.

I did the following:
1)	At the camera's webpage I activated ‘Edit Config -> Motion -> "Enable NVR Video stream: /sustain?video=1"’
2)	At the Home Assistant I added ‘Devices & Services -> Add integration -> Generic Camera: rtsp stream: http://IP_ADDRESS/sustain?video=1 and selected HTTP.’
3)	I added the "Frigate Lovelace Card" under HACS.
4)	In the Overview section I added the Frigate Lovelace Card and selected  the Generic Camera.

That’s it.

You don't need to install Frigate of Go2RTC or any other addidtional add-on.

Hope this helps and thanks a lot for the excellent work you are doing! 😊
